src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Text/LazyStringSplitTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+using Xunit;
+
+namespace Microsoft.VisualStudio.Text
+{
+    public sealed class LazyStringSplitTests
+    {
+        [Theory]
+        [InlineData("a;b;c", ';', new[] { "a", "b", "c" })]
+        [InlineData("a_b_c", '_', new[] { "a", "b", "c" })]
+        [InlineData("aa;bb;cc", ';', new[] { "aa", "bb", "cc" })]
+        [InlineData("aaa;bbb;ccc", ';', new[] { "aaa", "bbb", "ccc" })]
+        [InlineData(";a;b;c", ';', new[] { "a", "b", "c" })]
+        [InlineData("a;b;c;", ';', new[] { "a", "b", "c" })]
+        [InlineData(";a;b;c;", ';', new[] { "a", "b", "c" })]
+        [InlineData(";;a;;b;;c;;", ';', new[] { "a", "b", "c" })]
+        [InlineData("", ';', new string[0])]
+        [InlineData(";", ';', new string[0])]
+        [InlineData(";;", ';', new string[0])]
+        [InlineData(";;;", ';', new string[0])]
+        [InlineData(";;;a", ';', new[] { "a" })]
+        [InlineData("a;;;", ';', new[] { "a" })]
+        [InlineData(";a;;", ';', new[] { "a" })]
+        [InlineData(";;a;", ';', new[] { "a" })]
+        [InlineData("a", ';', new[] { "a" })]
+        [InlineData("aa", ';', new[] { "aa" })]
```

Comment:
Nit, I find it a little easier to view inputs/expectations if you turn this into table

Reply:
Fixed in 189d881616fe3dfead1d3861b6ab2645559251d8.